src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.ts

```diff
@@ -65,6 +75,16 @@ export class EditKeyboardMappingComponent implements OnInit {
     }
   }
 
+  public buildKeyCombo($event: string[]): void {
+    var currentValue: string[] = this.actionTriggerDataForm.get('keys')?.value ?? []
+
+    $event.forEach(item => {
+      if (item !== undefined && !currentValue.includes(item)) {
```

Comment:
When will `item` ever be undefined? 
`$event` is a `string[]`, so it's not possible for `$event` to contain an undefined value, hence this check is redundant.

Reply:
Concerns addressed and committed to git (but not yet pushed)